models/order.js

```diff
@@ -25,18 +25,11 @@ orderSchema.statics.getOrders = function() {
 orderSchema.statics.updateEntry = function(orderId, entryIndex, update) {
   return this.findOne({ id: orderId })
     .then((order) => {
```

Comment:
Yo good explanation of underscore vs ramda, also pCompose?? http://developer.telerik.com/featured/practical-functional-javascript-ramda/